brave-cxf3/README.md

```diff
@@ -65,7 +61,6 @@ CXF JAX-RS client setup:
 JAXRSClientFactoryBean clientFactory = new JAXRSClientFactoryBean();
 clientFactory.setAddress("http://localhost:9001/");
 clientFactory.setServiceClass(FooService.class);
-clientFactory.getInInterceptors().add(BraveClientInInterceptor.create(brave));
-clientFactory.getOutInterceptors().add(BraveClientOutInterceptor.create(brave));
+clientFactory.getFeatures().add(BraveClientFeature.create(brave));
```

Comment:
Add a section how to use TracerContext/CurrentSpanContext, especially when you would need to use it